src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -48,8 +51,9 @@ class ProfilerController
      * @param string                       $toolbarPosition The toolbar position (top, bottom, normal, or null -- use the configuration)
      * @param ContentSecurityPolicyHandler $cspHandler      The Content-Security-Policy handler
      * @param string                       $baseDir         The project root directory
+     * @param KernelInterface              $kernel          The kernel
```

Comment:
Do you think that it's this also the time to do this for the others optional aguments?